openquake/commonlib/logictree.py

```diff
@@ -792,6 +796,21 @@ def validate_uncertainty_value(self, node, branchset, value):
                 node, self.filename,
                 'expected a pair of floats separated by space'
             )
+        elif branchset.uncertainty_type == 'incrementalMFDAbsolute':
+            mbr = value.split(',')
+            if len(mbr) == 3:
+                min_mag, bin_width, rates = mbr
+                try:
+                    rates = positivefloat(rates)
```

Comment:
Should this be `valid.positivefloat`? Also, I would change the `except` clause to `except ValueError`.
Also, please run flake8 on this file, I get some warnings.